waku/waku_archive/driver/sqlite_driver/migrations.nim

```diff
@@ -14,12 +14,12 @@ logScope:
   topics = "waku archive migration"
 
 
-const SchemaVersion* = 7 # increase this when there is an update in the database schema
+const SchemaVersion* = 8 # increase this when there is an update in the database schema
 
 template projectRoot: string = currentSourcePath.rsplit(DirSep, 1)[0] / ".." / ".." / ".." / ".."
 const MessageStoreMigrationPath: string = projectRoot / "migrations" / "message_store"
 
-proc isSchemaVersion7*(db: SqliteDatabase): DatabaseResult[bool] =
+proc isSchemaVersion8*(db: SqliteDatabase): DatabaseResult[bool] =
```

Comment:
This proc is a temporary workaround that we will keep until version 0.30.0.
Therefore, we need to keep considering the hard-coded logic for the version 7 :D
```suggestion
proc isSchemaVersion7*(db: SqliteDatabase): DatabaseResult[bool] =
```